src/features/game/lib/processEvent.ts

```diff
@@ -122,8 +122,8 @@ export const MAX_ITEMS: Inventory = {
   Iron: new Decimal("800"),
   Stone: new Decimal("1600"),
   Wood: new Decimal("8000"),
-  "Wild Mushroom": new Decimal("100"),
-  "Magic Mushroom": new Decimal("100"),
+  "Wild Mushroom": new Decimal("75"),
```

Comment:
@eliasSFL OK, the wild mushrooms' limit was also decreased, following the same rule.

Reply:
@Droidze You can leave the wild mushroom at 100. cuz the BE for wild mushroom is at 150

Reply:
@eliasSFL I'm confused now, you said in your previous message wild mushrooms' hoard limit is 100 in the BE, so I've made changes for both of them. 
![image](https://github.com/user-attachments/assets/c7d1438b-3326-4995-aa61-74adcf726750)